### PR TITLE
Add scheduler-plugins Staging Registry

### PR DESCRIPTION
**Pull request by seanmalloy**
Creates a new gcr.io staging container registry for the SIG Scheduling
scheduler-plugins project.

https://github.com/kubernetes-sigs/scheduler-plugins

### CODE REVIEW

**Comment by seanmalloy** [comment]
@Huang-Wei and @denkensk as discussed in https://github.com/kubernetes-sigs/scheduler-plugins/issues/47 this PR creates the gcr.io container registry for the scheduler-plugins project. Let me know if you have any questions. Thanks!

/cc @Huang-Wei @denkensk

File: groups/groups.yaml [hunk 1]
```diff
@@ -1198,6 +1198,17 @@ groups:
...
+  - email-id: k8s-infra-staging-scheduler-plugins@kubernetes.io
+    name: k8s-infra-staging-scheduler-plugins
+    description: |-
+      ACL for staging scheduler-plugins
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - ahg@google.com
+      - wangqingcan1990@gmail.com
+      - wei.huang1@ibm.com
```
Comment (seanmalloy) [inline groups/groups.yaml:1210]: @Huang-Wei I believe the email addresses listed here need to be tied to a GSuite account(i.e. GMail). Please confirm this email is tied to a GSuite account.

**Comment by denkensk** [comment]
Thanks for your great work. @seanmalloy 
`k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.800`
I think it is better to keep the tag consistent with the release version of Kubernetes. 
like `k8s.gcr.io/scheduler-plugins/kube-scheduler:v1.18.8` ? WDYT? @Huang-Wei

**Comment by seanmalloy** [comment]
> Thanks for your great work. @seanmalloy 
> `k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.800`
> I think it is better to keep the tag consistent with the release version of Kubernetes. 
> like `k8s.gcr.io/scheduler-plugins/kube-scheduler:v1.18.8` ? WDYT? @Huang-Wei 

@denkensk this pull request only defines the the registry name `k8s.gcr.io/scheduler-plugins`.

The container image names and container image tags can be whatever you want. These will be determined by the build automation(i.e. Makefile) in the `kubernetes-sigs/scheduler-plugins` repo.

This PR is step one of a three step process.

**Comment by denkensk** [comment]
> > Thanks for your great work. @seanmalloy
> > `k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.800`
> > I think it is better to keep the tag consistent with the release version of Kubernetes.
> > like `k8s.gcr.io/scheduler-plugins/kube-scheduler:v1.18.8` ? WDYT? @Huang-Wei
> 
> @denkensk this pull request only defines the registry name `k8s.gcr.io/scheduler-plugins`.
> 
> The container image names and container image tags can be whatever you want. These will be determined by the build automation(i.e. Makefile) in the `kubernetes-sigs/scheduler-plugins` repo.
> 
> This PR is step one of a three step process.

----

Got it. Thanks ~ @seanmalloy

File: groups/groups.yaml (hunk 1 above)
Comment (Huang-Wei) [reply groups/groups.yaml:1210]: @seanmalloy Please update mine to `hweicdl@gmail.com`.

**Comment by Huang-Wei** [comment]
> Thanks for your great work. @seanmalloy
> `k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.800`
> I think it is better to keep the tag consistent with the release version of Kubernetes.
> like `k8s.gcr.io/scheduler-plugins/kube-scheduler:v1.18.8` ? WDYT? @Huang-Wei

Both ways have their pros and cons - let's continue the discussion in https://github.com/kubernetes-sigs/scheduler-plugins/issues/5.

File: groups/groups.yaml (hunk 1 above)
Comment (seanmalloy) [reply groups/groups.yaml]: @Huang-Wei fixed.

**Comment by seanmalloy** [comment]
@Huang-Wei and @denkensk I made one small adjustment to the accounts allowed to push to the registry. Can one of you add a /lgtm if you think it looks good? After that I will ask for a review/approval from `wg/k8s-infra`.

The PR also needs a /ok-to-test from an org member.

**Pull request merged**